deploy/cronjob/cronjob.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kube-janitor
+spec:
+  # run every minute
+  schedule: "*/1 * * * *"
```

Comment:
it probably makes sense to use a schedule with smaller frequency, e.g. every 5 minutes (I think the overhead of cronjobs makes the "run every minute" more fitting for the "while true" loop in a pod)

Reply:
Make sense to me. I translated your deployment example to a cronjob one. This is the reason why the cronjob was scheduled to run every minute. But i agree with the overhead behaviour.